sb-cb-ui-collection/library/sunbird-cb/collection/src/lib/ui-org-table/user-list/org-user-table.component.html

```diff
@@ -97,6 +99,10 @@
 <mat-menu #cardMenu="matMenu">
     <ng-template matMenuContent let-rowData="data">
         <!-- <div > -->
+        <button (click)="buttonClick('editInfo',rowData)" mat-menu-item name="channel-analytics-button"
+            class="ws-mat-primary-text" i18n-aria-label aria-label="Analytics">
+            <mat-icon>input</mat-icon> <span>Edit Info</span>

```

Comment:
Same icon as the other button? should add relevant icon which resembles the purpose of button